src/main/java/frc/robot/commands/OrganizeIndexCMD.java

```diff
@@ -27,15 +28,13 @@ public void initialize() {
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    if ( !(index.getBallPosition(1)) && index.getBallPosition(2) ){
-      isDone = false;
-      index.setSpeedTower(0.75);
-    } else {
-      isDone = true;
-    }
+
+
```

Comment:
If `execute()` is empty, then why does this command (still) exist?

Reply:
Deleted execute